_posts/2020-04-03-rendering-in-UE4.md

```diff
@@ -0,0 +1,32 @@
+---
+author: "alepaoletti"
+layout: post
+did: "blog10"
+title: "Rendering in Unreal Engine 4"
+slug: "RenderingInUE4"
+date: 2020-04-06 08:00:00
+categories: rendering
+img: /banners/GPU_profiler.png
+banner: /banners/GPU_profiler.png
+tags: graphics
+description: "A really quick overview of the rendering pipeline in Unreal Engine 4."
+---
+
+Unreal Engine 4 has a lot of code for doing rendering: it exists in its own module - the *Renderer* module - which is compiled to a dll to allow faster iteration. The *Renderer* module depends on the Engine module because it has many callbacks into Engine. However, when the Engine needs to call some code in the Renderer, this happens through the *IRendererModule* interface.
+
+The *RHI* module is the other key module for graphics programming as it is the interface for rendering APIs. Here is where the magic happens to allow the abstract rendering code to work on different platforms using different APIs.
```

Comment:
RHI first used here. Since this is an acronym, you should put the expansion between braces. e.g RHI (Rendering Hsometihng Isomething)